src/epics/helpers/ChatHandler.js

```diff
@@ -54,6 +55,7 @@ class ChatHandler {
           .map(JSON.parse)
           .do(contents => {
             this.wsMessageSubject.next({
+              id: guid.create(),
```

Comment:
@elimisteve FYI. In order to easier distinguish messages when deciding wether to scroll down or not i just create a unique ID locally until we have a better solution